src/darwin/Framework/CHIP/MTRDiagnosticLogsDownloader.mm

```diff
@@ -165,7 +165,6 @@ - (instancetype)initWithType:(MTRDiagnosticLogType)type
                     // data in the logs that the caller may find useful. For this reason, fileURL is passed in even
                     // when there is an error but fileHandle is not nil.
                     completion(strongSelf->_fileHandle ? fileURL : nil, bdxError);
-                    [strongSelf deleteFile];
```

Comment:
So what deletes the file and when?

Reply:
Agree with @bzbarsky-apple why is this removed?